pages/index.tsx

```diff
@@ -69,14 +69,17 @@ export default function Home(): ReactElement {
         <div>
           <a className="hover:text-gray-600" href="#chapter3">Quests</a>
         </div>
+        <div>
```

Comment:
I'm not sure why all these classes got reshuffled (perhaps the browser compat via `npx browserslist@latest --update-db`)?

This is the actual change.